examples/native-external-references/functions/helpers.spec.ts

```diff
@@ -51,7 +51,9 @@ describe('Getting the URLs', () => {
       lookupUrls: [
         'https://api.themoviedb.org/3/movie/urn:123?language=en-US',
         'https://api.themoviedb.org/3/movie/urn:456?language=en-US'
-      ]
+      ],
+      trendingUrl:
+        'https://api.themoviedb.org/3/trending/movie/day?language=en-US'
```

Comment:
Why is this addition needed?

Reply:
By default first search query is empty and TMDB responds with an empty array. To have _something_ displayed at first render of entity picker (before user starts typing the query), we've decided to pull results from the trending endpoint. We forgot to cover this in the example tests.